src/gui/torrentcontentmodel.cpp

```diff
@@ -235,7 +234,6 @@ void TorrentContentModel::updateFilesPriorities(const QVector<BitTorrent::Downlo
     if (m_filesIndex.size() != fprio.size())
         return;
 
-    emit layoutAboutToBeChanged();
```

Comment:
I wonder why there was no problem with checkbox updates in Qt5? Maybe this signal was a workaround?
But IIRC it should be accompanied by layoutChanged signal that is missing in corresponding places. Maybe revert notifySubtreeUpdated related changes and try to add layoutChanged in corresponding places?

Reply:
What are you trying to solve?

Reply:
> What are you trying to solve?

The same as #17786 does but in a different way, both w/o multiple dataChanged signal emitting and totally unrelated headerDataChanged signal.

Reply:
I can change notifySubtreeUpdated() to use `layoutChanged` instead of `headerDataChanged`.

Reply:
Replacing (in current master) `notifySubtreeUpdated` with `layoutChanged` and added `layoutAboutToBeChanged` in those places where there were no such signal emitted before `layoutChanged` works for me with Qt 6.4.

Reply:
Commit dropped.